src/js/populateBoard.js

```diff
@@ -0,0 +1,30 @@
+import { getLeaderBoard } from './API';
+
+function createElementBoard(leaderboardHTMLTag, scores) {
```

Comment:
- [ ] Please make use of the arrow function instead of the function keyword. For more information on how the ES6 arrow function syntax works, see this [Mozilla documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions).

Reply:
Thanks for your feedback @Ghiftee, It is now solved on commit f0d37df.